pipelines/h6c/rtp/v2/h6c_rtp_stage_1.toml

```diff
@@ -324,6 +324,7 @@ args = ["{basename}",
         "${A_PRIORI_STATUS_OPTS:good_statuses}"
         ]
 
+
```

Comment:
These won't run if they aren't also in the workflow above. Also, are you sure you don't want to pass any settings into them from the toml?

Reply:
There aren't any configurable settings yet beyond input filename.
I figured there was a step missing will add them.

Reply:
What about the names of the conda environments for running casa?

Reply:
Done